test/nodeMock/workAllocation/mockData.js

```diff
@@ -73,6 +73,56 @@ class WorkAllocationMockData {
         return tasks;
     }
 
+    getCaseList(count, actions) {
+        const caseActions = actions ? actions : [
+            { "id": "reassign", "title": "Reassign task" },
+            { "id": "unclaim", "title": "Release this task" }
+        ];
+        const cases = [];
+        for (let i = 0; i < count; i++) {
+            const taskDueDate = `"2021-02-16T18:58:48.987+0000"`;
+            cases.push({
```

Comment:
I _think_ you're returning tasks rather than cases here... 🤔

Reply:
Unused, so removed.

Reply:
The mockData will be removed shortly as @rayliangatos is retrieving actual data from the API

Reply:
Okay, cool.